bot/plotting/comparison_generator.py

```diff
@@ -79,9 +79,23 @@ def comparison_generator(
             param_list.append(params.copy())
 
             # generate a random value
-            param_value = parameter_value_generator(
-                chemistry, param_to_vary
-            )
+            while True:
+                param_value = parameter_value_generator(
+                    chemistry, param_to_vary
+                )
+                if (
+                    param_to_vary == "Electrode height [m]"
+                    or param_to_vary == "Electrode width [m]"
+                ):     # pragma: no cover
+                    if (
+                        param_value == 0
+                        or param_value < 0
```

Comment:
Is there any particular reason to not use `<=`?

Reply:
No particular reason, a silly mistake:)

Reply:
Technically it is not a mistake haha